eks/eks-cluster.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+    name: irosenba
```

Comment:
Can we use another name instead of irosenba?

Reply:
Yes, what name should I use?

Reply:
I don't have a good idea. Could be something like sample-config?